transcom/move.mil
# Feature/156211092/create menus module

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

- [x ] run the application locally (`make up`) and verified that my changes behave as expected.
- [x ] run static behat test suite (`circleci build --job behat`) against my changes.
- [x ] run the code sniffer (`circleci build --job code-sniffer`) against my changes.
- [x ] run the code coverage tool (`circleci build --job code-coverage`) 
      against my changes
- [x ] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x ] thoroughly outlined below the steps necessary to test my changes.
- [ x] attached screenshots illustrating relevant behavior before and after my changes.
- [x ] read, understand, and agree to the terms described in [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md).
- [x ] added my name, email address, and copyright date to [CONTRIBUTORS.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTORS.md).

## Summary of Changes

This pull request…

- Add the custom menu module. This module will add the menus to the configuration, facilitating the deployments. Now, every change of the menus needs to happen in the custom move mil menus module.
- Add contributing info relevant to menu updates in [CONTRIBUTING.md](https://github.com/Bixal/move.mil/blob/master/CONTRIBUTING.md) .
- Fix link in the pull request template from depofdefense repo to Bixal repo.

## Testing

To verify the changes proposed in this pull request…

1. Execute `make menu-update`
2. Go to [menu admin](http://move.mil.localhost:8000/admin/structure/menu) and verify the new menu items are there.

## Screenshots

<img width="1421" alt="main-menu" src="https://user-images.githubusercontent.com/37001835/37935172-9313e314-311e-11e8-9edd-ee6b0cf8d12b.png">
